packages/synapse/src/Synapse.test.ts

```diff
@@ -60,176 +59,192 @@ describe('When given the Synapse plugin', () => {
         },
       })
     })
-    test(
```

Comment:
These tests were not being applied properly. The async function needs to be passed into the test function as an argument.

Also the filter needed to be updated to use the $or pattern.

I already went ahead and made the required changes.